### PR TITLE
Add Service Usage Regional Example

### PR DESCRIPTION
**Pull request by upodroid**
Fixes: https://github.com/hashicorp/terraform-provider-google/issues/8369

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/third_party/terraform/website/docs/r/bigtable_instance.html.markdown [hunk 1]
```diff
@@ -9,9 +9,12 @@ description: |-
 
 # google_bigtable_instance
 
-Creates a Google Bigtable instance. For more information see
-[the official documentation](https://cloud.google.com/bigtable/) and
-[API](https://cloud.google.com/bigtable/docs/go/reference).
+Creates a Google Bigtable instance. For more information see:
+
+* [API documentation](https://cloud.google.com/bigtable/docs/reference/admin/rest/v2/projects.instances.clusters)
+* How-to Guides
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/r/bigtable_instance.html.markdown:15]: This should go in the composer docs

**Review by c2thorn — CHANGES_REQUESTED** [review]

File: mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+resource "google_service_usage_consumer_quota_override" "override" {
+  provider       = google-beta
+  dimensions = {
+    region = "us-central1"
+  }
+  project        = google_project.my_project.project_id
+  service        = "compute.googleapis.com"
+  metric         = "compute.googleapis.com%2Fn2_cpus"
```
Comment (c2thorn) [inline mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb:16]: Would the forward slash character not work here?

Comment (upodroid) [reply mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb:16]: Haven't tried, let me check

File: mmv1/third_party/terraform/website/docs/r/bigtable_instance.html.markdown (hunk 1 above)
Comment (upodroid) [reply mmv1/third_party/terraform/website/docs/r/bigtable_instance.html.markdown:15]: Fixed.

File: mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb (hunk 2 above)
Comment (upodroid) [reply mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb:16]: It doesn't work.


```
=== CONT  TestAccServiceUsageConsumerQuotaOverride_regionConsumerQuotaOverrideExample
2021/02/25 15:43:54 [WARN] Got error running Terraform: 
Error: Invalid escape sequence

  on ../../../../../../../../var/folders/b1/dthn83bs2qbcrg38qszm22440000gn/T/plugintest318460841/work555631348/config721254248/terraform_plugin_test.tf line 17, in resource "google_service_usage_consumer_quota_override" "override":
  17:   limit          = "\project\region"

The symbol "p" is not a valid escape sequence selector.

    provider_test.go:270: Step 1/1 error: Error running pre-apply refresh: 
        Error: Invalid escape sequence
        
          on ../../../../../../../../var/folders/b1/dthn83bs2qbcrg38qszm22440000gn/T/plugintest318460841/work555631348/config721254248/terraform_plugin_test.tf line 17, in resource "google_service_usage_consumer_quota_override" "override":
          17:   limit          = "\project\region"
        
        The symbol "p" is not a valid escape sequence selector.
        
--- FAIL: TestAccServiceUsageConsumerQuotaOverride_regionConsumerQuotaOverrideExample (14.29s)
FAIL
FAIL	github.com/hashicorp/terraform-provider-google-beta/v3/google-beta	16.778s
FAIL

```

Comment (c2thorn) [reply mmv1/templates/terraform/examples/region_consumer_quota_override.tf.erb:16]: Got it, thanks!

**Review by c2thorn — APPROVED** [review]

**Pull request merged**